### PR TITLE
added the named page property for chrome & firefox

### PR DESCRIPTION
**Pull request by dletorey**
#### Summary

Documenting named page property for @page at-rule

#### Test results and supporting details

Tested in the latest Firefox 110 and tested in Chrome.
The evidence for chrome version found [here](https://chromestatus.com/feature/5173237715566592)

#### Related issues

Content PR to come soon

### CODE REVIEW

**Review by queengooborg — DISMISSED** [review]

File: css/at-rules/page.json [hunk 1]
```diff
@@ -44,6 +44,45 @@
...
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": {
+                "version_added": "1.5"
+              },
+              "webview_android": {
+                "version_added": "37"
+              }
```
Comment (queengooborg) [inline css/at-rules/page.json:77]: ChromeStatus says this was implemented in WebView 85, and I'd be surprised that Samsung Internet supported this so long ago.

```suggestion
              "samsunginternet_android": "mirror",
              "webview_android": "mirror"
```

**Review by queengooborg — CHANGES_REQUESTED** [review]

**Comment by dletorey** [comment]
@queengooborg great spot

File: css/at-rules/page.json [hunk 2]
```diff
@@ -44,6 +44,41 @@
             "deprecated": false
           }
         },
+        "page": {
+          "__compat": {
+            "description": "<code>page</code> property",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@page/page",
```
Comment (estelle) [inline css/at-rules/page.json:50]:
```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/page",
```

this is a property, not a descriptor

Comment (estelle) [reply css/at-rules/page.json:50]: this should not be under the css/at-rules/page.json, as it's not an at rule

**Comment by estelle** [comment]
Is `@margins` anywhere? https://developer.mozilla.org/en-US/docs/Web/CSS/@page#margin_at-rules

File: css/at-rules/page.json (hunk 2 above)
Comment (dletorey) [reply css/at-rules/page.json]: Done

**Comment by dletorey** [comment]
@estelle the `@margins` at-rule is not currently supported by any browser, so [in accordance with this](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#features-with-no-browser-support) I have not added it (well actually I added it and then removed it).

**Review by queengooborg — APPROVED** [review]

**Pull request merged**